packages/firestore/src/core/version.ts

```diff
@@ -15,7 +15,5 @@
  * limitations under the License.
  */
 
-import firebase from '@firebase/app';
-
 /** The semver (www.semver.org) version of the SDK. */
-export const SDK_VERSION = firebase.SDK_VERSION;
+export { version as SDK_VERSION } from '../../../firebase/package.json';
```

Comment:
You will need to use 'rollup-plugin-json' to resolve it in all build tasks.
You will want to set `preferConst: true` for es2017 builds.

Reply:
I am already using it in exp and lite, and we have been using it in the legacy build already. Is there some other target that is missing?

Reply:
My mistake. I did see it in `rollup.config.es5.js`, but now I realize es5 builds are based on esm2017 builds which uses it.